frameworks/hdfs/src/main/dist/krb5.conf

```diff
@@ -6,8 +6,9 @@
 [realms]
   {{SECURITY_KERBEROS_REALM}} = {
     kdc = {{SECURITY_KERBEROS_KDC_HOSTNAME}}:{{SECURITY_KERBEROS_KDC_PORT}}
+    admin_server = {{SECURITY_KERBEROS_KDC_HOSTNAME}}:{{SECURITY_KERBEROS_KDC_PORT}}
```

Comment:
Are any of the other settings required here? https://web.mit.edu/kerberos/krb5-1.12/doc/admin/conf_files/krb5_conf.html#realms